newscout_web/frontend/src/app/DashboardApp/Article.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import moment from 'moment';
+import ReactDOM from 'react-dom';
+import { ToastContainer } from 'react-toastify';
+import * as serviceWorker from './serviceWorker';
+import {ARTICLE_LIST_URL, ARTICLE_STATUS_URL} from '../../utils/Constants';
+import DashboardMenu from '../../components/DashboardMenu';
+import DashboardHeader from '../../components/DashboardHeader';
+import { getRequest, postRequest } from '../../utils/Utils';
+import { Button, Form, Input, Row, Col, Table } from 'reactstrap';
+
+import './index.css';
+
+class Article extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			results: [],
+			page : 0,
+			next: null,
+			previous: null,
+			loading: false,
+			q: "",
+			articleUpdateId: ""
+		};
+	}
+
+	getArticles = () => {
+		var url = ARTICLE_LIST_URL;
+		getRequest(url, this.setArticleData);
```

Comment:
See if this aspect can be swapped with [React Hooks](https://reactjs.org/docs/hooks-intro.html)

Reply:
completed with traditional reactjs callbacks.